docs/workflows/quick-start-guide.md

```diff
@@ -1,52 +1,131 @@
-```
-g3t clone aced-demo
-```
+* Similar to `git init`, this command initializes a new project in the current directory
+* The id is `aced-demo`; the program name is `aced`, the project name is `demo`
+* `Program name` is predefined by the institution, the `g3t ping` command will show the available programs
+* `Project name` must be unique within the server, and must be alphanumeric and contain no spaces
+
+
+### Add file(s):
 
-Add file(s)
 ```
 g3t add folder/file.tsv
 g3t add folder/file2.tsv
 ```
 
-Create meta data for your files. This `g3t` command will take care of creating this for you
+* This command will record an entry for each file in the `MANIFEST` directory.
+* Optionally you can tag each file with patient, specimen or other identifiers. See `g3t add --help` for more information.
+
+
+### Create meta data for your files. The `g3t meta init` command will take care of creating FHIR compliant data for you.
+
 ```
-g3t utilities meta create
+g3t meta init
 ```
 
-Check that the expected files are queued for upload
+* This command will create metadata files in the META directory. At a minimum, the directory will contain:
+
+| file                     | contents                   |
+|--------------------------|----------------------------|
+| ResearchStudy.ndjson     | Description of the project |
+| DocumentReference.ndjson | File information           |
+
+Additional metadata files for patient, specimen, and other entities will be generated based on options provided to the `add` command.
+
+| file                   | contents               |
+|------------------------|------------------------|
+| Patient.ndjson         | Patient information    |
+| ResearchSubject.ndjson | Enrollment information |
+| Specimen.ndjson        | Sample information     |
+
+
+> Note: You may supply your own FHIR data, the `g3t meta init` command is a convenience method to generate the required files.
+
+### Check that the metadata is valid:
+
+```
+g3t meta validate
+```
+
+
+### Check that the expected files are queued for upload
 ```
 g3t status
 ```
 
-Commit files
+### Commit files
 ```
 g3t commit -m "Adding files"
 ```
 
-Push to ACED IDP
+### Push to ACED IDP
 ```
 g3t push
 ```
 
-View the upload status (pending or complete)
+Notes:
+* a push will fail if the metadata is not valid or if the files are not committed.
+* a push will fail a data file was already uploaded, use the `--overwrite` option to force an upload.
+
+There are several other options available for the `push` command for specialized use cases. See `g3t push --help` for more information.
+
+### View the upload status (pending or complete)
 ```
 g3t status
 ```
 
 Check that your data was uploaded successfully by viewing on the Exploration tab on [aced-idp.org](https://aced-idp.org)
 
-## 2. Download Data
+## 2. Cloning a project
+
+```commandline
+g3t clone [OPTIONS] PROJECT_ID 
+```
+
+The clone command will download the MANIFEST/ and META/ directories associated with the project. 
```

Comment:
maybe...
> associated with the project _into its own directory_

Reply:
also, I'm not able to use env var G3T_PROFILE here as well, had to se a profile.
```
WARNING: No --profile specified, found multiple gen3_client profiles: ['local', 'dev', 'staging', 'cbds-prod', 'cbds-dev']
INFO: No config file found in current directory or parents.
'NoneType' object has no attribute 'strip'
```

Shall we include --profile flag in the example command?

Reply:
Works for me: ❓ 

```$ G3T_PROFILE=local g3t clone cbds-testgrip
Cloning -[2024-10-07 18:33:05,425][   INFO] File .g3t/work/cbds-testgrip.git.zip downloaded successfully
Cloned .g3t/work/cbds-testgrip.git.zip
On branch main
nothing to commit, working tree clean
```

Reply:
forgot to export, my bad